src/System.Drawing.Common/tests/Printing/MarginsTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
@mellinoe I'm curious: is this some special case when using Mono code? Mono's code is MIT too.